Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM ruby:2.7.0-alpine
 
 RUN apk add --update --no-cache  \
   build-base \
+  git \
```

Comment:
is this necessary?

Reply:
@jcoyne It was when I was pointing DIA at a GitHub branch. I figured I'd leave it in to make it easier to do this in the future.

Reply:
But this makes the images substantially bigger, which means pulling this image for testing takes longer.

Reply:
@jcoyne looks like it adds ~16MB to a package that is ~155MB. :man_shrugging: I can remove it.

Reply:
10% reduction!

Reply:
Removed